assembl/tests/views_tests/test_graphql.py

```diff
@@ -2622,3 +2624,18 @@ def test_get_all_posts(graphql_request, proposition_id):
     first_post = res.data['posts']['edges'][0]['node']
     assert proposition_id == first_post['id']
     return res
+
+def test_dogpile_cache():
```

Comment:
Move this test out of `test_graphql`. It's not a GraphQL test, per se. It's a test of the dogpile cache being successful. You can create a new test module by creating a new test file, `test_caching.py`, perhaps under `lib_tests`. And put this test there. You'll have to make your imports accordingly there.